include/miopen/miopen.h

```diff
@@ -6577,11 +6577,139 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendDestroyDescriptor(miopenBackendDescrip
 MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t descriptor,
                                                      miopenBackendDescriptorType_t descriptorType,
                                                      size_t sizeInBytes);
-
 /** @} */
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+/** @addtogroup LossFunction
+ * @{
+ */
+
+/*! MSELoss APIs
+ * @brief MSELoss forward functon
+ *
+ * MSELoss forward, reduced function
+ * @param  [in]  handle         An instance of miopenHandle_t
+ * @param  [in]  xDesc          Input Tensor descriptor
+ * @param  [in]  yDesc          Target Tensor descriptor
+ * @param  [in]  zDesc          Output Tensor descriptor
+ * @param  [in]  x              Pointer to input tensor data
+ * @param  [in]  y              Pointer to target tensor data
+ * @param  [out] z              Pointer to the output tensor data
+ * @param  [out]  workspace      Pointer to the workspace area (for reduction)
```

Comment:
Please match the code indentation.

Reply:
I’ve updated the indentation to match others